REFACTORING_CANDIDATES.md

```
@@ -0,0 +1,266 @@
+1. **space_manager.rs (1,402L)** → can split into 6 focused modules (~200-250L each)
+2. **world_entry.rs (932L)** → can split into 5-6 message handler modules (~150-200L each)
+3. **dispatch.rs (662L)** → **should NOT split** (already clean architecture)
+
+---
+
+## Detailed File Structure
+
+### Cell Service: `crates/services/src/cell/`
+
+**Current:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)